todo_app/flask_config.py

```diff
@@ -13,3 +13,5 @@ def __init__(self):
         self.CLIENT_SECRET = os.environ.get('CLIENT_SECRET')
         self.STATE = os.environ.get('STATE')
         self.LOGIN_DISABLED = os.getenv('LOGIN_DISABLED') == 'True'
+        self.LOG_LEVEL = os.environ.get('LOG_LEVEL')
+        self.LOGGLY_TOKEN = os.environ.get('LOGGLY_TOKEN')
```

Comment:
The main is a bit ahead of this `exercise-13` branch. Terraform code is missing in this PR. You will need to update your terraform and make sure it runs, hopefully before merging. Make sure this branch is ahead of `main` by rebasing or merging and update any terraform related setup with new Loggly values